add 0.61.0 notes (#4057)
weaveworks/eksctl: docs/release_notes/0.61.0.md

# Release 0.61.0
## Features

- Add ability to set a custom subnet when creating a nodegroup (#4025 #4044)

## Bug Fixes

- Set new selector session when creating nodegroup (#3993)

## Acknowledgments

> Weaveworks would like to sincerely thank:

Reviewer: shall we remove this?

Author: if https://github.com/weaveworks/eksctl/pull/4056 gets merged we will need to add in this person :smile: 

but otherwise yeah you are right we should remove it :smile:

Reviewer: nevermind! it'll be needed when #4056 is merged!

# Release 0.61.0
## Features

> - Add ability to set a custom subnet when creating a nodegroup (#4025 #4044)

Reviewer 2: ```suggestion
- Add ability to set a custom subnet when creating a nodegroup (#4025 #4044)
- Expose nodegroup container runtime configuration (#4051)
```
got this in under the wire

# Release 0.61.0
## Features

- Add ability to set a custom subnet when creating a nodegroup (#4025 #4044)

## Bug Fixes

> - Set new selector session when creating nodegroup (#3993)

Reviewer 3: WDYT about this?
```suggestion
- Fix panic when creating a nodegroup with instance selector options
```

> - Return message when VPC config missing on unowned ng create (#4056)

Reviewer 3: Since this is a bug fix:
```suggestion
- Fix panic when VPC config is missing (#4056)
```

- Fix panic when VPC config is missing (#4056)

## Acknowledgments
Weaveworks would like to sincerely thank:

>   @NaivePa

Reviewer 2: ```suggestion
  @NaivePa and @SharpThunder 
```
